applications/skyepass.md

```diff
@@ -184,46 +184,22 @@ Besides private work for companies that cannot be shared, Song developed a simpl
 | 3.     | Client Side UI/UX | an simple Electron UI/UX not wired up with logic yet         |
 
 
+### Milestone 1  — Desktop App & Browser Extension 
```

Comment:
Personally, I don't particularly mind this grant remaining open (at an appropriate price), though in the last discussion we had I think the consensus was rather that it should be closed, i.e. terminated in mutual agreement, if you're gonna work on something else. Doing so would also allow you more freedom when you've finished the other grant, and to be honest if your plans have changed in the last few months it's perfectly possible and fine if they do again.

Reply:
Sure. We can come to agreement on this. We can replace the application for SkyePass to the SkyeKiwi Protocol. The reason that we are doing the replacement is actually a very rational decision: we realize that the secret transmitting technology underneath is more valuable than a password manager and for other project to reuse. For the next 12 month: we are building the protocol out and assemble a blockchain network. No switching now. Haha

Reply:
Whichever you prefer. I'm actually happy to accept this as is, let's see what the rest of the committee thinks. And in that case I agree with you ;)